src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -379,6 +379,7 @@ void ImageReplayer<I>::bootstrap() {
 
   {
     Mutex::Locker locker(m_lock);
+    request->get();
```

Comment:
It will not work for mock (unittest) classes. Also, ImageReplayer has long live, and can be restarted.

Before the change the request objects already had a property that they are automatically destroyed after send() is finished. I wan to keep this property and just need a way to control (delay) this destruction to be sure cancel() is called for yet existing object.